app/src/main/res/values/strings.xml

```diff
@@ -324,6 +324,7 @@
     <string name="msg_wait">Please wait…</string>
     <string name="msg_copied">Copied to clipboard</string>
     <string name="no_available_dir">Please define a download folder later in settings</string>
```

Comment:
Should I remove this now that we're not using this anymore?

Reply:
Which one?
Please wait is being used [here](https://github.com/wb9688/NewPipe/blob/use-saf-by-default/app/src/main/java/us/shandian/giga/ui/adapter/MissionAdapter.java#L982)
Copied to clipboard [here](https://github.com/wb9688/NewPipe/blob/use-saf-by-default/app/src/main/java/org/schabi/newpipe/util/ShareUtils.java#L104) and [here](https://github.com/wb9688/NewPipe/blob/use-saf-by-default/app/src/main/java/us/shandian/giga/util/Utility.java#L212)
Please define … [here](https://github.com/wb9688/NewPipe/blob/use-saf-by-default/app/src/main/java/org/schabi/newpipe/settings/DownloadSettingsFragment.java#L278) and [here](https://github.com/wb9688/NewPipe/blob/use-saf-by-default/app/src/main/java/us/shandian/giga/service/DownloadManagerService.java#L566)

Reply:
>no_available_dir here and here

You're right, sorry, though I don't think that's the most appropriate string to use in those places.